pysrc/models/column_statistics.py

```diff
@@ -0,0 +1,17 @@
+from dataclasses import dataclass
+from dataclasses_json import dataclass_json
+
+from .column_information import ColumnInformation
+
+@dataclass_json
+@dataclass(frozen=True)
+class ColumnStatistic:
+    
+    column_Information: ColumnInformation
+    count: int
+    unique_count: int
+    unique_ratio: float
+    min: str
+    max: str
```

Comment:
I'm still not 100% sure about those stats. When running it on SCOP, you get the following stats for a column of the baseline:

```json
                {
                    "column_Information": {
                        "table_name": "DESCRIPTION",
                        "column_name": "column1"
                    },
                    "count": 94439,
                    "unique_count": 94439,
                    "unique_ratio": 1.0,
                    "min": "100000",
                    "max": "99999",
                    "shortest": "51926",
                    "longest": "100000"
                }
```

I'm not sure the semantics of `min` and `max` is ideal for numeric columns, but I think it would be good enough for now and we should rather update these when we actually need them. So, in my opinion, you can do either way here (update the statistics now or do it only later when it might be needed).

Reply:
For strings this is absolutely useless...

It just says '1' < '9' so '10000' < '9999'.

In the future we could do something with Levenshtein Distance or something like that...